### DISCUSSION TITLE
Feedback form - how to activate

### DISCUSSION THREAD

**Discussion by johan-almlof** [squidfunk/mkdocs-material#3810] [Q&A] [answered]
We are having trouble activating the feedback function.
1. Code added to mkdocs.yml as per instructions.
2. Google analytics is configured as well.

What is the next step? What, if any code, must be added to an article so that the feedback question appears?
BR/Johan

**Comment by squidfunk** [comment]
Are you using Insiders? The feedback widget is currently only in Insiders. Could you please share your configuration? In general, no code must be added, so when you configure the feedback as said via `mkdocs.yml`, data should flow into your Google Analytics account. Also, make sure to go through the __How to visualize the collected feedback ratings?__ instructions.

**Reply by squidfunk** [reply]
Ah, one more thing – please make sure you're on the latest version of Insiders. I'm not sure __how__ you are using Insiders (there are multiple installation methods), but there was a bug where the feedback rating disappeared some days ago.

This was fixed in [`215d63e`](https://github.com/squidfunk/mkdocs-material-insiders/commit/215d63ef0b52fb9d8599b80e6f89c271001a4bf4).

**Comment by johan-almlof** [comment]
Hello Martin.

Thank you. We are on insiders. We will check the versions.

Any ideas or suggestions if we need to add any code or commands to individual articles? In short, should feedback icons appears on all pages and articles in my help center or only in those I assign? And if so, how do I assign which pages to display?

BR/Johan

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Reply by squidfunk** [reply]
> Any ideas or suggestions if we need to add any code or commands to individual articles? In short, should feedback icons appears on all pages and articles in my help center or only in those I assign? And if so, how do I assign which pages to display?

The configuration in `mkdocs.yml` + Google Analytic setup is all that is needed, nothing needs to be added to the articles. Feedback will appear on all pages, except those [explicitly disabled](https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-analytics/#hiding-the-feedback-widget).

**Reply by squidfunk** [reply]
Any update? Did you make it work?

**Comment by johan-almlof** [comment, marked as answer]
Hi Martin. Yes. Thanks. We made it work. Next week we will look to deploy on external site.

We updated sw.
What unlocked it was the syntax in the mkdocs file.

I appreciate the follow up.
Happy easter/Johan

**Reply by squidfunk** [reply]
Great to hear!
